### PR TITLE
add notifications sign to mango create account process

### PR DESCRIPTION
**Pull request by abrzezinski94**

### CODE REVIEW

**Comment by saml33** [comment]
I'm not sure we should do this as you can't even create an account on Mango with a Ledger atm

**Comment by abrzezinski94** [comment]
> I'm not sure we should do this as you can't even create an account on Mango with a Ledger atm

oh ok so ill remove the isledger thing and just leave sign msg to process

**Comment by abrzezinski94** [comment]
ledger check removed

**Review by saml33 — APPROVED** [review]

File: components/account/CreateAccountForm.tsx [hunk 1]
```diff
@@ -28,26 +30,27 @@ const getNextAccountNumber = (accounts: MangoAccount[]): number => {
...
 }: {
-  isFirstAccount?: boolean
   customClose?: () => void
   handleBack?: () => void
 }) => {
   const { t } = useTranslation('common')
   const [loading, setLoading] = useState(false)
   const [name, setName] = useState('')
-  const { wallet } = useWallet()
+  const [singToNotifications, setSignToNotifications] = useState(true)
```
Comment (tlrsssss) [inline components/account/CreateAccountForm.tsx:42]: nitpick: "sign" is misspelled here

File: components/modals/UserSetupModal.tsx [hunk 2]
```diff
@@ -65,13 +66,17 @@ const UserSetupModal = ({
   const [depositAmount, setDepositAmount] = useState('')
   const [submitDeposit, setSubmitDeposit] = useState(false)
   const [sizePercentage, setSizePercentage] = useState('')
+  const [singToNotifications, setSignToNotifications] = useState(true)
```
Comment (tlrsssss) [inline components/modals/UserSetupModal.tsx:69]: same typo here

**Review by tlrsssss — CHANGES_REQUESTED** [review]
Added a nitpic regarding a type.

This can be added in a followup but I think it would be useful to have a little info icon with a tooltip on the UserSetupModal explaining what the notifications are for.

**Pull request merged**